src/NuGet.Core/NuGet.Commands/GlobalSuppressions.cs

```diff
@@ -216,3 +216,6 @@
 [assembly: SuppressMessage("Build", "CA1307:The behavior of 'string.GetHashCode()' could vary based on the current user's locale settings. Replace this call in 'Microsoft.Extensions.Internal.HashCodeCombiner.Add(string)' with a call to 'string.GetHashCode(System.StringComparison)'.", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.Extensions.Internal.HashCodeCombiner.Add(System.String)")]
 [assembly: SuppressMessage("ApiDesign", "RS0026:Do not add multiple public overloads with optional parameters", Justification = "API already shipped", Scope = "member", Target = "~M:NuGet.Commands.RestoreSummary.Log(NuGet.Common.ILogger,System.Collections.Generic.IReadOnlyList{NuGet.Commands.RestoreSummary},System.Boolean)")]
 [assembly: SuppressMessage("Globalization", "CA1307:Specify StringComparison", Justification = "Annoying and dumb", Scope = "member", Target = "~M:NuGet.Commands.UnresolvedMessages.GetMessageAsync(System.String,NuGet.LibraryModel.LibraryRange,System.Collections.Generic.IList{NuGet.DependencyResolver.IRemoteDependencyProvider},System.Boolean,System.Collections.Generic.IList{NuGet.DependencyResolver.IRemoteDependencyProvider},NuGet.Protocol.Core.Types.SourceCacheContext,NuGet.Common.ILogger,System.Threading.CancellationToken)~System.Threading.Tasks.Task{NuGet.Common.RestoreLogMessage}")]
+[assembly: SuppressMessage("Globalization", "CA1305:Specify IFormatProvider", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.Extensions.FileSystemGlobbing.Internal.MatcherContext.CombinePath(System.String,System.String)~System.String")]
```

Comment:
If adding more suppression, then please fill out `Justification` for why we need this.